src/cs/production/Flecs/flecs.cs

```diff
@@ -7960,7 +7960,7 @@ public struct ecs_xtor_t
         public const int PINVOKE_TARGET_ENV_MSVC = 1;
 
         [CNode(Kind = "MacroObject")]
-        public static string PINVOKE_TARGET_NAME = "aarch64-pc-windows-msvc";
+        public static string PINVOKE_TARGET_NAME = "x86_64-pc-windows-msvc";
```

Comment:
This needs to get fixed in `CAstFFi` merge option. `PINVOKE_TARGET_NAME` should not be added to bindgen because it has different values among different compilers.